docs/data-tools/arm-forge.md

```diff
@@ -0,0 +1,238 @@
+## Arm Forge
+
+[Arm Forge](https://developer.arm.com/Tools%20and%20Software/Arm%20Forge)
+provides debugging and profiling tools for MPI parallel applications, and
+OpenMP or pthreads mutli-threaded applications (and also hydrid MPI/OpenMP).
+The debugger and profiler are called DDT and MAP, respectively.
+
+ARCHER2 has a license for up to 64 nodes (8192 cores) shared between
+all users at any one time. (Note cores are counted by the license, not
+MPI processes, threads, or any other software entity.)
+
+There are two ways of running the Arm user interface. If you have a good
+internet connection to ARCHER2, the GUI can be run on the front-end (with
+an X-connection).
+Alternatively, one can download a copy of the Forge remote client to your
+laptop or desktop, and run it locally. The remote client may be the
+preferred method for many users.
```

Comment:
Be a bit stronger here: e.g. "We strongly recommend that users run the remote client on their laptop/desktop.